video/rest/compositionhooks/list-hooks/meta.json

```diff
@@ -0,0 +1,4 @@
+{
+  "title": "Create a Composition Hook mixing all room audio tracks",
```

Comment:
The title is wrong, should be `List all enabled Composition Hooks of an account`